moveit_py/package.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>moveit_py</name>
+  <version>0.0.0</version>
+  <description>Python binding for MoveIt 2</description>
+
+  <maintainer email="peterdavidfagan@gmail.com">Peter David Fagan</maintainer>
+
+  <license>BSD</license>
+
+  <author email="peterdavidfagan@gmail.com">Peter David Fagan</author>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <build_depend>pybind11_vendor</build_depend>
+
+  <exec_depend>ament_index_python</exec_depend>
+
+  <depend>rclcpp</depend>
+  <depend>rclpy</depend>
+  <depend>geometry_msgs</depend>
+  <depend>octomap_msgs</depend>
+  <depend>moveit_ros_planning_interface</depend>
+  <depend>moveit_ros_planning</depend>
+  <depend>moveit_core</depend>
+  <depend>jupyter-notebook</depend>
```

Comment:
Why do we have this as a dependency? I think it should be moved to the tutorials package

Reply:
Yes this is correct it is an exec_depend for the tutorial.

My thinking on adding it as a library dependency was that a user may want to open a notebook independent of the tutorials to interact with the Python library. If we don't install notebooks by default we expect the user to manually install the notebook Python package. I thought adding it as a dependency of the library made sense given I saw notebooks as a key tool for quickly interacting with the Python API. 

It shouldn't be a depend though for sure, probably exec_depend is more appropriate in the library package.xml specification. What are your thoughts?

Reply:
I think if it's not a direct dependency, it shouldn't be listed in the package.xml. But it's just a nit

Reply:
Yeah this fair, I'll move it.

Reply:
Ok I dropped the commit.